Project-3/Jvonderh Project 3

```diff
@@ -0,0 +1,220 @@
+File 2 is input of Computer 3
+Computer 3 receives input from Computer 1
+Computer 3 is an instance_of material entity
+Computer 1 2 and 3 are part of Network 1
+Network 1 is an instance_of Network
+Network is_a object aggregate
+
+
+6. Using Protege, place these in the BFO hierarchy where you think they fit best:
+```
+  (a) Bach's Well-Tempered Clavier
+  Generically Dependent Continuant
+  (b) Chair of the UB Philosophy Department
+  Role
+  (c) SARS-CoV-2
+  Object
+  (d) Mexico City
+  Object Aggregate
+  (e) The trunk of a minivan
+  Fiat Object Part
+  (f) Occupation
+  Role
+  (g) Ocean
+  Object Aggregate
+  (h) Lake
+  Object Aggregate
+```
+
+7. True or False; explain your answers:
+```
+  (a) An instance of Material Entity can have an instance of Immaterial Entity as part.
+  True. A material entity has some portion of matter as proper or improper continuant part. These parts don't need to be material. For example thoughts are immaterial but can be understood as a part of the material person. 
+  (b) An instance of Immaterial Entity can have an instance of Material Entity as part.
+  False. An immaterial entity is immaterial, so it may be concretized in a material entity or otherwise but may not have it as part. For example, a thought may be an immaterial entity which has a material entity as a subject, but that subject is either not material (i.e is a representation of a material thing) or is not a part of that thought.
+  (c) An organization may have another organization as part.
+  True. Organizations break off separate organizations to perform different functions all of the time. A charity organization for finding housing for the homeless might break off local and regional organizations to better serve people at various locations for example.
+  (d) An organization may have no members as part. 
+  True. Organizations can continue their existence even after losing all members for a time. For example, the Soviet Union continued to exist for a time even after all member states left the organization. It's continuation, then, depended not on its members but instead on other factors like the organizational structure and rules.
+  (e) Any site is partially bounded by some instance of Material Entity.
+  True. All examples of sites used in the BFO Protege consist of gaps between material entities. Gaps such as this, while separate from their surrounding material entities, will be partially defined by those material entities. For example, the Grand Canyon is specifically a gap defined by surrounding cliffs.
+  (f) A book placed under the leg of a wobbly table has acquired a new function. 
+  False. The book may have been primarily used to read or write in, but it was always a function of it that it could take up space and act as a solid base for another object. Function is not limited by its purpose of creation, and instead includes its ability to hold up a table or any other such object by virtue of its physical make-up.
```

Comment:
Different from disposition or capacity, An entity's function is indeed limited by its purpose of being created or naturally selected. Capacity include Function, but not vice versa.

Reply:
I don't want to rehash the conversation Finn and I had on the Google Doc here, but I think it is legitimate to interpret function as it is laid out in the BFO in this way. The point about it being intentionally created or naturally developed explains how something got the physical features, but it is the physical features themselves which determine the function. If there is another source to view on this let me know, but I believe my reading here is accurate to the Elucidation of function.